easybuild/easyconfigs/p/Primer3/Primer3-2.5.0-foss-2020a.eb

```
@@ -0,0 +1,17 @@
+name = 'Primer3'
+version = '2.5.0'
+
+homepage = 'https://primer3.org/'
+description = """Primer3 is a widely used program for designing PCR primers (PCR = 'Polymerase Chain Reaction').
+ PCR is an essential and ubiquitous tool in genetics and molecular biology.
+ Primer3 can also design hybridization probes and sequencing primers."""
+
+toolchain = {'name': 'foss', 'version': '2020a'}
+
+source_urls = ['https://github.com/primer3-org/%(namelower)s/archive/refs/tags/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['7581e2fa3228ef0ee1ffa427b2aa0a18fc635d561208327471daf59d1b804da0']
+
+runtest = 'test'
+
```

```suggestion

sanity_check_commands = [
    'primer3_masker -h',
    # https://github.com/biobuilds/biobuilds/issues/21
    'cd %(builddir)s/%(namelower)s-%(version)s/src && primer3_core ../example'
]
```

Restore the `sanity_check_commands` from `Primer3-2.4.0-iccifort-2019.1.144-GCC-8.2.0-2.31.1.eb`.